cli/client.go

```diff
@@ -940,9 +996,7 @@ func (c *viamClient) robotParts(orgStr, locStr, robotStr string) ([]*apppb.Robot
 }
 
 func (c *viamClient) printRobotPartLogsInner(logs []*commonpb.LogEntry, indent string) {
-	// Iterate over logs in reverse because they are returned in
-	// order of latest to oldest but we should print from oldest -> newest
```

Comment:
@zaporter-work , I no longer do _any_ obvious reversing of logs. They seem to print newest->oldest when I run against one of my robots... I sense that my new batching + page token strategy removes the need to invert anything. Feel free to run against one of your own robots to see if output looks as expected.

Reply:
I feel fairly strongly that it should print oldest - > newest. 

It feels really foreign / confusing to have the logs printed in newest -> oldest

Reply:
My comment was just indicating that perhaps some of the reordering logic would fit better here in the robotPartLogs fn (but maybe not)

Reply:
@dgottlieb Do you have an opinion on this?

Reply:
I do also agree that oldest -> newest would be a better experience

Reply:
> I feel fairly strongly that it should print oldest - > newest.
It feels really foreign / confusing to have the logs printed in my terminal as newest -> oldest

I'm of the same opinion.

Reply:
Oh whoops I think I just misread some previous comments and thought newest->oldest was desired. I'll switch back!

Reply:
Switched back to oldest->newest. The current location of the inversion makes sense to me, and I think your existing comment is fine @zaporter-work.